README.md

```diff
@@ -1,173 +1,185 @@
 # Obstacle Detection and Avoidance
-PX4 computer vision algorithms packaged as ROS nodes for depth sensor fusion and obstacle avoidance. This repository contains two different implementations, targeting different use cases:
+PX4 computer vision algorithms packaged as ROS nodes for depth sensor fusion and obstacle avoidance. This repository contains two different implementations:
 
-  * A local, VFH+ based planner that plans (including some history) in a vector field histogram
-  * A global, graph based planner that plans in a traditional occupancy grid
+  * *local_planner* is a local VFH+* based planner that plans (including some history) in a vector field histogram
+  * *global_planner* is a global, graph based planner that plans in a traditional octomap occupancy grid
+
+The two algorithms are standalone and they are not meant to be used together.
+
+The *local_planner* requires less computational power but it doesn't compute optimal paths towards the goal since it doesn't store information about the expolred environment. On the other hand, the *global_planner* buils a map of the envirnment leading to optima
   
-**The full PX4 Developer documentation is available on: https://dev.px4.io and includes more computer vision (including VIO) examples.**
+The documentation contains information about how to setup and run the two planner systems on the Gazebo simulator and on a companion computer running Ubuntu 16.04, for both avoidance and collision prevention use cases.
+
+> **Note** PX4-side setup is covered in the PX4 User Guide:
+  - [Ostacle Avoidance](https://docs.px4.io/en/computer_vision/obstacle_avoidance.html)
```

Comment:
These two bullets aren't rendering as part of the note. Perhaps put a > at start of both lines.

Typo in link: "Obstacle"